### PR TITLE
spdlog: add first recipe

### PR DESCRIPTION
**Pull request by hmenn**
* Now, spdlog library is enable under recipes-support. Just install with "spdlog-dev".
* It's also tested with populate-sdk.

### CODE REVIEW

File: meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+DESCRIPTION = "Very fast, header only, C++ logging library."
+HOMEPAGE = "https://github.com/gabime/spdlog/wiki"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/files/common-licenses/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+PR="r0"
```
Comment (kraj) [inline meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb]: We dont need this. Remove it.

File: meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+SRC_URI = "git://github.com/gabime/spdlog.git;protocol=git;branch=v1.x;tag=v${PV}"
```
Comment (kraj) [inline meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb]: No tags please. use hardcoded git SHA, tags are floating and dont work in NO_NETWORK situations

File: meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb (hunk 1 above)
Comment (hmenn) [reply meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb]: done

File: meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb (hunk 2 above)
Comment (hmenn) [reply meta-oe/recipes-support/spdlog/spdlog_1.3.1.bb]: done

**Comment by hmenn** [comment]
All requests were completed.

**Comment by hmenn** [comment]
@kraj Commit was pushed into master-next before changes. We need to merge this pr also for requested changes.

**Pull request closed without merging**